types/level/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-outside-dependencies": false
```

Comment:
please move this inline `// tslint:disable .....`, so you could put a comment why this is needed